I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 187, in script
    show kitchen with dissolve
IOError: Couldn't find file 'music/MGS.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 187, in script
    show kitchen with dissolve
  File "F:\ren\renpy-7.2.2-sdk\renpy\ast.py", line 1375, in execute
    renpy.exports.with_statement(trans, paired)
  File "F:\ren\renpy-7.2.2-sdk\renpy\exports.py", line 1578, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "F:\ren\renpy-7.2.2-sdk\renpy\display\core.py", line 2240, in do_with
    clear=clear)
  File "F:\ren\renpy-7.2.2-sdk\renpy\display\core.py", line 2690, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "F:\ren\renpy-7.2.2-sdk\renpy\display\core.py", line 3360, in interact_core
    renpy.audio.audio.periodic()
  File "F:\ren\renpy-7.2.2-sdk\renpy\audio\audio.py", line 1019, in periodic
    six.reraise(exc[0], exc[1], exc[2])
  File "F:\ren\renpy-7.2.2-sdk\renpy\audio\audio.py", line 997, in periodic_thread_main
    periodic_pass()
  File "F:\ren\renpy-7.2.2-sdk\renpy\audio\audio.py", line 938, in periodic_pass
    c.periodic()
  File "F:\ren\renpy-7.2.2-sdk\renpy\audio\audio.py", line 420, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "F:\ren\renpy-7.2.2-sdk\renpy\audio\audio.py", line 72, in load
    rv = renpy.loader.load(fn)
  File "F:\ren\renpy-7.2.2-sdk\renpy\loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'music/MGS.mp3'.

Windows-8-6.2.9200
Ren'Py 7.2.2.491
test1 1.0
Sat Jun 08 04:00:45 2019
